pom.xml

```diff
@@ -15,7 +15,7 @@
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Promoted+Builds+Plugin</url>
 
   <properties>
-    <jenkins.version>1.565</jenkins.version>
+    <jenkins.version>1.614</jenkins.version>
```

Comment:
It must be an LTS baseline version at least (e.g. 1.625.3).
java.level 6 in such case won't work since Java 7 is the new minimal version starting from 1.610